app/code/Magento/Checkout/Test/Mftf/Test/StorefrontAddProductWithAllTypesOfCustomOptionToTheShoppingCartTest.xml

```diff
@@ -105,7 +105,7 @@
         <!-- Assert Product Count in Mini Cart -->
         <actionGroup ref="StorefrontAssertMiniCartItemCountActionGroup" stepKey="assertProductCountAndTextInMiniCart">
             <argument name="productCount" value="2"/>
-            <argument name="productCountText" value="2 Item in Cart"/>
+            <argument name="productCountText" value="2 Items in Cart"/>
         </actionGroup>
 
```

Comment:
@ihor-sviziev during test i just find out this issue. We can cover and test this case too

Reply:
Only visual issue not actual big problem

Reply:
I believe adjusting this MFTF test will be enough, but just wanted to make sure QA team will double check it manually in order to make sure there no regression issues.